mllib/src/main/scala/org/apache/spark/mllib/stat/Statistics.scala

```diff
@@ -212,4 +213,15 @@ object Statistics {
     : KolmogorovSmirnovTestResult = {
     KolmogorovSmirnovTest.testOneSample(data, distName, params: _*)
   }
+
+  /** Java-friendly version of [[kolmogorovSmirnovTest()]] */
+  @varargs
+  def kolmogorovSmirnovTest(
+      data: JavaRDD[java.lang.Double],
+      distName: String,
+      params: java.lang.Double*): KolmogorovSmirnovTestResult = {
+    val javaParams = params.toSeq.asInstanceOf[Seq[Double]]
```

Comment:
Does the cast actually manage to change the objects into scala doubles or does enough magic happen that it works out? a test might be handy here.